client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityClient.java

```diff
@@ -407,6 +409,35 @@ public DeleteRoleMappingResponse deleteRoleMapping(DeleteRoleMappingRequest requ
                 DeleteRoleMappingResponse::fromXContent, emptySet());
     }
 
+    /**
+     * Retrieves roles in the native realm.
```

Comment:
IMO s/native realm/native roles store/
(I know this comment uses the lingo from the docs)